ui/vault-reporting/declarations/components/vault-reporting/connect-cluster.d.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
```

Comment:
Hmm not sure why you have this declaration file for this. I think this is from an old test branch that never got merged so maybe the sync script needs to empty out the dis/declaration folders before generating.